prusti-tests/tests/typecheck/fail/time-resources/condition.rs

```diff
@@ -0,0 +1,14 @@
+// compile-flags: -Ptime_reasoning=true
+
+use prusti_contracts::*;
+
+// ignore-test: prusti-constract functions used outside specification check not yet implemented
```

Comment:
I'm not sure about "not yet" here: we don't want to implement something like this. Many specification functions don't make sense in executable code, and branching on `time_credits` or `receipts` is also such an example. I would remove this test.

Reply:
As they don't make sense in executable code why don't we report them when we find them?

Reply:
Because nobody has implemented that feature yet...